<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Receiptionist</class>
 <widget class="QDialog" name="Receiptionist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Phone No.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Occupation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>148</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="txt_FName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_Sname"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_Pno">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_occupation"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>90</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Residence</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Religion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Patient Registration</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>98</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Relation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Email Address</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>98</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Occupation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Phone No.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLineEdit" name="txt_Goccupation"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_GPno">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Guardian Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDateEdit" name="txt_DOB">
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_residence"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_religion"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>261</y>
     <width>148</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLineEdit" name="txt_GName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_GRelation"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_email">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>117</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Patient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>145</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Register New Patient</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
